generators/client/templates/react/package.json

```diff
@@ -5,21 +5,21 @@
     "@fortawesome/react-fontawesome": "0.1.16",
     "@reduxjs/toolkit": "1.6.2",
     "axios": "0.24.0",
-    "bootstrap": "4.6.1",
+    "bootstrap": "5.1.3",
     "bootswatch": "5.1.3",
     "lodash": "4.17.21",
     "path-browserify": "1.0.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-hook-form": "7.19.1",
-    "react-jhipster": "0.17.0",
+    "react-jhipster": "github:jhipster/react-jhipster#main",
```

Comment:
This is temporary right?

Reply:
Sure, once a new version is released, we switch to it.